eng/pipelines/compliance.yml

```diff
@@ -56,6 +56,23 @@ jobs:
       uploadTSLint: false
       uploadAsync: true
 
+  ### Run latest version of APIScan listed at https://www.1eswiki.com/wiki/APIScan_Build_Task
+  - task: APIScan@2
+    displayName: Run APIScan
+    inputs:
+      softwareFolder: "$(Build.Repository.LocalPath)\\artifacts\\symbolstoindex"
```

Comment:
@kartheekp-ms @zivkan is it correct path for symbol's folder?

Reply:
We should download the artifacts into the new pipeline. As Andy mentioned  in https://github.com/NuGet/NuGet.Client/pull/4048#discussion_r631405674

Reply:
Can you check now if I'm doing it correctly?